gx_suite_definitions/proteomics_tmt.ipynb

```diff
@@ -0,0 +1,303 @@
+    "validator.expect_column_values_to_be_unique(\"uniqid\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# hgnc_symbol\n",
+    "validator.expect_column_values_to_be_of_type(\"hgnc_symbol\", \"str\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"hgnc_symbol\", 1, 15)\n",
+    "validator.expect_column_values_to_match_regex(\"hgnc_symbol\", \"^[a-zA-Z0-9.-]*$\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# uniprotid\n",
+    "validator.expect_column_values_to_be_of_type(\"uniprotid\", \"str\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"uniprotid\", 1, 15)\n",
+    "validator.expect_column_values_to_match_regex(\"uniprotid\", \"^[a-zA-Z0-9.-]*$\")\n",
+    "validator.expect_column_values_to_be_unique(\"uniprotid\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ensembl_gene_id\n",
+    "validator.expect_column_values_to_be_of_type(\"ensembl_gene_id\", \"str\")\n",
+    "validator.expect_column_values_to_not_be_null(\"ensembl_gene_id\")\n",
+    "validator.expect_column_value_lengths_to_equal(\"ensembl_gene_id\", 15)\n",
+    "# checks format and allowed chatacters\n",
+    "validator.expect_column_values_to_match_regex(\"ensembl_gene_id\", \"^ENSG\\d{11}$\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# tissue\n",
+    "validator.expect_column_values_to_be_of_type(\"tissue\", \"str\")\n",
+    "validator.expect_column_value_lengths_to_be_between(\"tissue\", 1, 15)\n",
+    "validator.expect_column_values_to_be_in_set(\"tissue\", [\"AntPFC\", \"DLPFC\", \"MFG\", \"TCX\"])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# log2_fc\n",
+    "validator.expect_column_values_to_be_of_type(\"log2_fc\", \"float\")\n",
+    "validator.expect_column_values_to_be_between(\"log2_fc\", -0.5, 1.5)\n",
+    "validator.expect_column_values_to_not_be_null(\"log2_fc\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ci_upr\n",
+    "validator.expect_column_values_to_be_of_type(\"ci_upr\", \"float\")\n",
+    "validator.expect_column_values_to_be_between(\"ci_upr\", -0.5, 2)\n",
+    "validator.expect_column_values_to_not_be_null(\"ci_upr\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ci_upr\n",
+    "validator.expect_column_values_to_be_of_type(\"ci_upr\", \"float\")\n",
+    "validator.expect_column_values_to_be_between(\"ci_upr\", -0.5, 2)\n",
+    "validator.expect_column_values_to_not_be_null(\"ci_upr\")"
```

Comment:
Is this chunk of code a duplicate of the one above it?

Reply:
Sure is, I guess I copy-pasted and didn't realize at some point! Fixed in my latest commit.